ldk

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+command=$1
+params=${@:2}
+cur_dir_name=${PWD##*/}
+project_dir="/var/www/$cur_dir_name"
+script=$(readlink -f "$0")
+script_dir=$(dirname $script)
+
+node_init="source \"\$NVM_DIR/nvm.sh\" && export PATH=\$PATH:/home/laradock/.nvm/versions/node/\$(echo \"nvm current\")/bin"
```

Comment:
@appleboy No, no! This script runing on host but nvm installed into container!
node_init should execute inside container, without it you can't use node at all!